db/dcrpg/pgblockchain.go

```diff
@@ -1277,236 +1287,356 @@ func (pgb *ChainDB) ticketPoolVisualization(interval dbtypes.TimeBasedGrouping)
 	return
 }
 
-// retrieveDevBalance retrieves a new DevFundBalance without regard to the cache
-func (pgb *ChainDB) retrieveDevBalance() (*DevFundBalance, error) {
-	bb, hash, err := pgb.HeightHashDB()
+func (pgb *ChainDB) updateProjectFundCache() error {
+	// Update non-merged rows.
+	_, _, err := pgb.AddressHistory(pgb.devAddress, 1, 0, dbtypes.AddrTxnAll)
```

Comment:
AddressHistory is used to update both balance and rows.

Reply:
In #1053, these calls to `AddressHistory` should be replaced by `AddressBalance` and two `AddressRows` calls.